.github/workflows/vale.yml

```diff
@@ -0,0 +1,24 @@
+name: Linting Markdown with Vale
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Lint Markdown with Vale
+        uses: errata-ai/vale-action@master
+        with:
+          styles: |
+            https://github.com/errata-ai/Microsoft/releases/latest/download/Microsoft.zip
+            https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
+          config: https://raw.githubusercontent.com/ory/docs/e66b0701ef8c4d18c32d2488ac905cb54e23f581/docs/.vale.ini
```

Comment:
You can probably `${{ github.sha }}` here

Reply:
works great thanks.